steering_vectors/aggregators.py

```diff
@@ -30,6 +53,23 @@ def pca_aggregator(pos_acts: Tensor, neg_acts: Tensor) -> Tensor:
     return sign * vec
 
 
+def _get_normalized_regression_coef(
+    pos_acts: Tensor, neg_acts: Tensor, regression: Regression
+) -> Tensor:
+    reg = regression.fit(
+        torch.cat([pos_acts, neg_acts]).cpu().to(torch.float32).numpy(),
```

Comment:
I'm surprised this works. Don't we need to center each pos/neg pair so they're 0 mean? Or do something like we do in PCA, and use positive and negative versions of the deltas as the positive and negative samples rather than using the raw samples themselves?

Reply:
> Don't we need to center each pos/neg pair so they're 0 mean?
Yes, it turns out that this is actually necessary to get the best results, as I learned when I updated the unit test 

Done with [7ba358f](https://github.com/steering-vectors/steering-vectors/pull/22/commits/7ba358f8229cbe662932a7d35cffe92cefbf6e28)